src/datastore/pg-store.ts

```diff
@@ -2846,6 +2846,20 @@ export class PgStore {
     return { found: true, result: queryResult };
   }
 
+  async getRawEventCount(): Promise<number> {
+    try {
+      const result = await this.sql<
+        Promise<number>[]
+      >`SELECT count(*) from event_observer_requests`;
```

Comment:
Since you just need this for testing I think you could simply perform this query directly inside the test:
```typescript
const result = await db.sql<{ count: number }[]>`SELECT count(*) from event_observer_requests`;
```
The reason I say this is because COUNTs are extremely expensive so we should not facilitate their use in our regular codebase

Reply:
got it